elasticsearch/Dockerfile

```diff
@@ -31,7 +31,8 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
     yum clean all
 # initial boring elasticsearch.yml is just to enable plugin installation.
 # put the real config in place in a deployment with a ConfigMap.
-ADD elasticsearch.yml logging.yml /usr/share/elasticsearch/config/
+ADD elasticsearch.yml /usr/share/elasticsearch/config/
+ADD index_templates /usr/share/elasticsearch/config/index_templates
```

Comment:
@richm Wouldn't it be better to do `shopt -o failglob` in the beginning of the script instead before the loop? (May this can clash with some assumptions and break existing logic in the original script? Or is there any reason why it is good to do it just before the loop?)